libs/auth-api-lib/src/lib/services/personalRepresentative.service.ts

```diff
@@ -49,14 +45,14 @@ export class PersonalRepresentativeService {
     const result = await paginate({
       Model: this.personalRepresentativeModel,
       limit: query.limit || 10,
-      after: query.after,
-      before: query.before,
+      after: query.after ?? '',
+      before: query.before ?? '',
```

Comment:
Interesting. This is mildly annoying.

Reply:
Yes there is a mismatch in the PaginationDto class and the PaginateInput interface so this is needed unless we change the interface